packages/web-pkg/src/composables/clipboard/useClipboard.ts

```diff
@@ -0,0 +1,27 @@
+import { useClipboard as _useClipboard } from '@vueuse/core'
+
+export const useClipboard = () => {
+  // doCopy creates the requested link and copies the url to the clipboard,
+  // the copy action uses the clipboard // clipboardItem api to work around the webkit limitations.
+  //
+  // https://developer.apple.com/forums/thread/691873
+  //
+  // if those apis not available (or like in firefox behind dom.events.asyncClipboard.clipboardItem)
+  // it has a fallback to the vue-use implementation.
+  //
+  // https://webkit.org/blog/10855/
+  const copyToClipboard = (quickLinkUrl: string) => {
```

Comment:
Could you please check if we still need this, we have so many code paths where we just use the browser native code

Reply:
@fschade recently added this via https://github.com/owncloud/web/pull/9832 because of some Safari-foo.

Reply:
I mean the else case. Just checked that and navigation.clipboard was introduced 5years ago to all recent browsers. We don't use the useClipboard from vue-use elsewhere. I think we could kick this legacy else case

Reply:
But the else-case is not legacy, FF e.g. needs it according to Flo's comment.

Either way, if changes are needed here let's do it in another PR because it's just moved code and nothing I want to be touching here.

Reply:
I will investigate if I have some free time